aiohttp/connector.py

```diff
@@ -58,18 +66,15 @@
     ssl = None  # type: ignore[assignment]
     SSLContext = object  # type: ignore[misc,assignment]
 
-
```

Comment:
Could you exclude the formatting changes from the PR? It's a bad tone to mix them with any behavior changes.

Reply:
Aplogies, these are caused by the IDE. I will revert it.